src/components/Layout.jsx

```diff
@@ -0,0 +1,27 @@
+import { Link, Outlet } from "react-router-dom";
+import style from "../styles/Layout.module.css";
+const Layout = () => (
+  <div className={style.container}>
+    <div className={style.layout}>
+      <h1>Math Magicians</h1>
+      <div className="navigation">
+        <Link className={`${style.link} ${style.navdecoration}`} to="/">
+          Home
+        </Link>
+        <Link
+          className={`${style.link} ${style.navdecoration}`}
+          to="/calculator"
+        >
+          Calculator
+        </Link>
+        <Link className={style.link} to="/quotes">
+          Quotes
+        </Link>
+      </div>
+    </div>
+    <div>
+      <Outlet />
+    </div>
+  </div>
+);
```

Comment:
- [ Required ] I think it would be better if you fix this error from `react`, currently anytime I run your application I encounter the error below. I would like to suggest that you change the imports in this file to use `single` quotes and not `double` quotes as shown below.

```JavaScript
import { Link, Outlet } from 'react-router-dom';
import style from '../styles/Layout.module.css';

const Layout = () => (
  <div className={style.container}>
    <div className={style.layout}>
      <h1>Math Magicians</h1>
      <div className="navigation">
        <Link className={`${style.link} ${style.navdecoration}`} to="/">
          Home
        </Link>
        <Link
          className={`${style.link} ${style.navdecoration}`}
          to="/calculator"
        >
          Calculator
        </Link>
        <Link className={style.link} to="/quotes">
          Quotes
        </Link>
      </div>
    </div>
    <div>
      <Outlet />
    </div>
  </div>
);
export default Layout;
```

## Screenshot of Error
![Screenshot from 2023-07-22 03-15-08](https://github.com/Samuelwanza/mathmagicians/assets/16392046/f7e6e1c7-fa78-4b2b-bd7c-8ed53d99646e)